res_users_with_gravatar/README.rst

```diff
@@ -0,0 +1,33 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Synchronize Gravatar image
+==========================
+
+Use you Gravatar image in to your Odoo
+
+
+License
+=======
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published
+by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/agpl-3.0-standalone.html>.
+
+
+Credits
+=======
+
+Contributors
+------------
+
+* Endika Iglesias <endika2@gmail.com>
```

Comment:
Please follow this template https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst